__tests__/app.test.js

```diff
@@ -187,4 +187,76 @@ describe("all tests", () => {
         });
     });
   });
+
+  describe("7-POST/api/reviews/:review_id/comments", () => {
+    it("should accept valid username, and body and respond 201 status", () => {
+      return request(app)
+        .post("/api/reviews/1/comments")
+        .send({
+          username: "mallionaire",
+          body: "Epic game!",
+        })
+        .expect(201);
```

Comment:
The ticket says that we should be responding with the given comment, so we should test we're getting it back.